addons/web/static/src/scss/modal.scss

```diff
@@ -114,7 +114,6 @@
 
     > .modal {
         display: block;
-        pointer-events: none;
```

Comment:
You see that this is re-enabled for modal-content below, so here either you remove something important, either you leave dead code behind :/ Probably dead code? 

Actually this code seems to disable the possibility to click on the .modal element itself but you now bind event handlers on the modal itself right?... I am lost: I thought bootstrap needed the backdrop to detect the closing click but in fact there is indeed the modal on top without this Odoo custom code... so they in fact only used the backdrop for the color?

Reply:
No It s also used for the click to close de modal. For owl dialog I moved the onclick on the modal instead of the backdrop. So I need to remove this pointer-events. And you are right the following pointer-event is dead code.
Im also lost, dialogs are a mess ...